# 4R robot pose defined as (x, y, z) position and end-affector orientation in the 3R chain's 3d plane

float32[3] position # [mm]
float32 orientation # [rad]
